public/v4/apps/droppy.yml

```diff
@@ -20,6 +20,6 @@ caproverOneClickApp:
             droppy is deployed and available as $$cap_appname.
 
             IMPORTANT: It will take up to 2 minutes for droppy to be ready. Before that, you might see 502 error page.
-    displayName: ''
+    displayName: 'droppy [archived]'
```

Comment:
archived?

Reply:
Droppy has been [archived](https://github.com/silverwind/droppy) and is not maintain anymore since few years 
It use node 12 and may introduce some security failure.
As the owner explain itself that the project is archived 
>Development on droppy has ceased because I don't have enough time or motivation to properly support it and because of its outdated technology stack, it became exceedingly boring to work.

I add this tag but I would be happy to write it anywhere if there is a way to specify this information in any part of the yml file

Reply:
(in the about we can read : **ARCHIVED** Self-hosted file storage)

Reply:
This seems to be a different droppy then? https://github.com/droppyjs/droppy

Reply:
You are totally righ I'll fix that right now